mars/core/operand/shuffle.py

```diff
@@ -31,6 +31,11 @@ class MapReduceOperand(Operand):
     mapper_id = Int32Field("mapper_id", default=0)
     # for reducer
     reducer_index = TupleField("reducer_index", FieldTypes.uint64)
+    # Total reducer nums, which also be shuffle blocks for single mapper.
+    n_reducer = TupleField("n_reducer", FieldTypes.uint64)
```

Comment:
Why not `Int64Field `?

Reply:
`n_reducer` is a non-negative number

Reply:
But, it's not match type of the `n_reducer` in `BaggingSample`:
```python
    n_reducer: int = Int64Field("n_reducer", default=None)
```
Also, here is a `TupleField` not an `UInt64Field`.

Reply:
Good catch, I'll change it to `Int64Field `